docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -567,18 +619,18 @@ This specification brings the following advantages:
   interfering with the entries of other installed operating systems.
 
 * Drop-in directories are now pretty ubiquitous on Linux as an easy way to
-  extend configuration without having to edit, regenerate or manipulate
+  extend boot loader menus without having to edit, regenerate or manipulate
   configuration files. For the sake of uniformity, we should do the same for
   the boot menu.
 
-* Userspace code can sanely parse boot loader configuration which is essential
+* Userspace code can sanely parse boot loader menu entries which is essential
   with modern firmware which does not necessarily initialize USB keyboards
   during boot, which makes boot menus hard to reach for the user. If userspace
-  code can parse the boot loader configuration too, UI can be written that
+  code can parse the boot loader menu entries too, UI can be written that
   select a boot menu item to boot into before rebooting the machine, thus not
   requiring interactivity during early boot.
 
-* To unify and thus simplify configuration of the various boot loaders, which
+* To unify and thus simplify menu entries of the various boot loaders, which
```

Comment:
Also here, the sentence made more sense before. The resulting menu entries are about the same, just how we derive them is changed.

Reply:
to me loader.conf is boot loader *configuration*, but this spec is not about loader.conf, that's not even mentioned. Instead here we focus on menu entry descriptions, which aren't really *configuration*